packages/xchain-mayachain-amm/TRADE_OPERATIONS.md

```
@@ -0,0 +1,92 @@
+      amount: new AssetCryptoAmount(
+        assetToBase(assetAmount('0.1')), 
+        assetFromStringEx('ETH.ETH')
+      ),
+      address: 'maya1...', // Your Maya address
+    })
+    console.log('Transaction hash:', result.hash)
+  } catch (error) {
+    console.error('Error:', error.message)
+  }
+}
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)